packages/graph/deployment/docker-compose.yml

```
@@ -4,7 +4,7 @@ volumes:
     external: true
 services:
   postgres:
-    image: postgres:14-alpine
+    build: postgres:14-alpine
```

I think this can be changed back as we don't have a `Dockerfile` anymore?
```suggestion
    image: postgres:14-alpine
```